projects/nlp/junit_testing.py

```diff
@@ -130,97 +151,23 @@ def _testModel(args, model, testData, labelRefs):
         pass
       degreesOfSeperation[degree] = separation / float(count) if count else None
 
-    if args.verbosity > 0:
+    if verbosity > 0:
       print
       print "Doc {}: {}".format(docId, wrapper.fill(document))
       print "Min, mean, max of ranks = {}, {}, {}".format(
-        overallRanks.min(), overallRanks.mean(), overallRanks.max())
+        ranks.min(), ranks.mean(), ranks.max())
```

Comment:
This will generate a lot of printouts but from the code and printouts it's hard to tell what we expect to happen and whether the test "succeeded".  If ranks.max() is more than 4 or 5 (depending on the test) then that particular test "failed" right?  Should we print out something specifically for that case?